Modules/_zstd/zstddict.c

```diff
@@ -124,23 +124,33 @@ PyDoc_STRVAR(ZstdDict_dictid_doc,
 "The special value '0' means a 'raw content' dictionary,"
 "without any restrictions on format or content.");
 
-PyDoc_STRVAR(ZstdDict_dictcontent_doc,
-"The content of a Zstandard dictionary, as a bytes object.");
-
 static PyObject *
 ZstdDict_str(PyObject *ob)
 {
     ZstdDict *dict = ZstdDict_CAST(ob);
     return PyUnicode_FromFormat("<ZstdDict dict_id=%u dict_size=%zd>",
-                                dict->dict_id, Py_SIZE(dict->dict_content));
+                                dict->dict_id, dict->dict_len);
```

Comment:
```suggestion
                                (unsigned int)dict->dict_id, dict->dict_len);
```

Reply:
dict_id is already a uint32_t, is there a reason for adding this cast?

Reply:
On every supported platform int is 32-bit, but it is not harm to be pedantic. This just goes to show that we didn't miss it.